codespell_lib/_codespell.py

```diff
@@ -28,6 +28,11 @@
 import textwrap
 
 word_regex_def = u"[\\w\\-'’`]+"
+# While we want to treat characters like ( or " as okay for a starting break,
+# these may occur unescaped in URIs, and so we are more restrictive on the
```

Comment:
Ah I guess my later comment is because of this?

Reply:
If you mean the `"http://foo"abandonned` example, yes.